GCMessengerSDKSample/app/src/main/AndroidManifest.xml

```diff
@@ -1,34 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
-
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"
+        android:maxSdkVersion="32" />
```

Comment:
This is correct to define it?

Reply:
This is not related to the targetsdk. This permission is just deprecated after sdk version 32 so had to add that attribute